.gitignore

```diff
@@ -12,6 +12,7 @@
 # next.js
 /.next/
 /out/
+/.contentlayer/
```

Comment:
We're not using ContentLayer anymore so I think we don't need this. You might still have a contentlayer folder on your machine from before the architecture refactor, but you can remove it.

Reply:
okay, thanks for the info. removed `.contentLayer` from `.gitignore`